paper/src/main/java/com/programmerdan/minecraft/simpleadminhacks/hacks/BetterVehicles.java

```diff
@@ -0,0 +1,517 @@
+package com.programmerdan.minecraft.simpleadminhacks.hacks;
+
+import com.destroystokyo.paper.event.entity.EntityRemoveFromWorldEvent;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.reflect.TypeToken;
+import com.programmerdan.minecraft.simpleadminhacks.SimpleAdminHacks;
+import com.programmerdan.minecraft.simpleadminhacks.configs.BetterVehiclesConfig;
+import com.programmerdan.minecraft.simpleadminhacks.framework.SimpleHack;
+import net.kyori.adventure.text.Component;
+import org.bukkit.*;
```

Comment:
* imports bad

Reply:
"Imports bad" as in star import bad or as in all imports bad? You gotta be more specific than "bad" please.

Reply:
\* I think if you put a backslash in front it doesn't make it a list.